NCARDDatabase/settings.py

```diff
@@ -14,6 +14,15 @@
 import os
 import sys
 
+from datetime import timedelta
+
+AUTO_LOGOUT = {
+    'IDLE_TIME': timedelta(minutes=10),
+    # 'SESSION_TIME': timedelta(minutes=30),
```

Comment:
Just delete this line instead of leaving it commented out

Reply:
delete it